keda/templates/01-serviceaccount.yaml

```diff
@@ -11,8 +11,8 @@ metadata:
   {{- if or .Values.podIdentity.azureWorkload.enabled .Values.serviceAccount.annotations }}
   annotations:
     {{- if .Values.podIdentity.azureWorkload.enabled }}
-    azure.workload.identity/client-id: {{ .Values.podIdentity.azureWorkload.clientId }}
-    azure.workload.identity/tenant-id: {{ .Values.podIdentity.azureWorkload.tenantId }}
+    azure.workload.identity/client-id: {{ .Values.podIdentity.azureWorkload.clientId | quote }}
+    azure.workload.identity/tenant-id: {{ .Values.podIdentity.azureWorkload.tenantId | quote }}
```

Comment:
@v-shenoy I hope this is OK for you?

Reply:
I already added those before you merged.

Reply:
Oh you're adding it to the client and tenant ids, whoops. Yeah this is fine, I think.

Reply:
Not for all of them, hence why I'm doing those here as well